src/coreclr/vm/field.h

```diff
@@ -285,6 +285,14 @@ class FieldDesc
         SetOffset(FIELD_OFFSET_NEW_ENC);
     }
 
+    BOOL IsCollectible()    
+    {
+        LIMITED_METHOD_DAC_CONTRACT;
+
+        LoaderAllocator *pLoaderAllocatorOfMethod = GetApproxEnclosingMethodTable()->GetLoaderAllocator();
+        return pLoaderAllocatorOfMethod->IsCollectible();
```

Comment:
```suggestion
        LoaderAllocator *pLoaderAllocator = GetApproxEnclosingMethodTable()->GetLoaderAllocator();
        return pLoaderAllocator->IsCollectible();
```
This is a field, not a method.

Reply:
Nice catch; copy-paste